linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/rule.yml

```diff
@@ -22,8 +22,6 @@ rationale: |-
 
 severity: medium
 
-platform: machine  # The oscap sysctl probe doesn't support offline mode
```

Comment:
Limitation of OpenSCAP. Content needs to be tool agnostic.

Reply:
Kernel-related rules don't make sense in offline mode. Everything kerne-related is host's responsibility.

Reply:
Agree with @matejak here, you cannot modify kernel parameters from a container, definitely not in the default configuration. And if it is allowed, it is kind of shooting into your foot as you are modifying the kernel of the underlying host OS which can have some unexpected (read unpleasant) consequences.